### ISSUE TITLE
Mapzen

### ISSUE THREAD

**Issue by gaitgait** [PocketMobsters/MobBotTemp#111]
I have a Mapzen elevation key but when I use it within the config (so set to true and using the key). I get "() Found old Altitude data, using this to avoid Mapzen API call.". Does this mean that this function does not work yet?

**Comment by gaitgait** [comment]
Guess I didnt move my location, so it just uses the old one

**Comment by pehur** [comment]
yes, it reads the inside the folder "MapzenAPI" if the current location is equal to config location.
